packages/nitro-protocol/contracts/ninja-nitro/AdjudicatorFactory.sol

```diff
@@ -40,8 +40,8 @@ contract AdjudicatorFactory {
     }
 
     /// @dev Allows us to create new channel contract and get it all set up in one transaction
-    function createChannel(bytes32 channelId) public returns (address payable channel) {
-        channel = payable(_deployChannelProxy(channelId));
+    function createChannel(bytes32 channelId) public {
```

Comment:
For my understanding: N-N doesn't require the return address because the contract address is computable.